bigchaindb/commands/bigchaindb.py

```diff
@@ -317,41 +315,38 @@ def create_parser():
                                help='The backend to use. It can only be '
                                '"localmongodb", currently.')
 
-    # parser for managing validator elections
-    validator_parser = subparsers.add_parser('upsert-validator',
-                                             help='Add/update/delete a validator.')
-
-    validator_subparser = validator_parser.add_subparsers(title='Action',
-                                                          dest='action')
-
-    new_election_parser = validator_subparser.add_parser('new',
-                                                         help='Calls a new election.')
+    # parser for managing elections
+    election_parser = subparsers.add_parser('election',
+                                            help='Manage elections.')
 
-    new_election_parser.add_argument('public_key',
-                                     help='Public key of the validator to be added/updated/removed.')
+    election_subparser = election_parser.add_subparsers(title='Action',
+                                                        dest='action')
 
-    new_election_parser.add_argument('power',
-                                     type=int,
-                                     help='The proposed power for the validator. '
-                                          'Setting to 0 will remove the validator.')
+    new_election_parser = election_subparser.add_parser('new',
+                                                        help='Calls a new election.')
 
-    new_election_parser.add_argument('node_id',
-                                     help='The node_id of the validator.')
+    new_election_subparser = new_election_parser.add_subparsers(title='Election_Type',
+                                                                dest='election_type')
 
-    new_election_parser.add_argument('--private-key',
-                                     dest='sk',
-                                     help='Path to the private key of the election initiator.')
+    # Parser factory for each type of new election, so we get a bunch of commands that look like this:
+    # election new <some_election_type> <args>...
+    for name, data in elections.items():
+        help = data['help']
```

Comment:
`help` is a reserved keyword - I would just `.add_parser(name, data['help'])`.

Reply:
Fixed